.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+npm run type-check -- --incremental false
```

Comment:
Why is the modifier needed, don't we want to run it incrementally so it's faster?